index.js

```diff
@@ -12,11 +12,10 @@ module.exports = {
   // ],
 
   locals(options) {
-    let name = options.entity.name;
-    let component = componentize(name);
+    let name = componentize(options.entity.name);
```

Comment:
There are no tests in this repo, no.  I generally test locally while making changes with `ember new foo -b ../path/to/this/repo`